frontend/package.json

```diff
@@ -36,7 +38,6 @@
     ]
   },
   "devDependencies": {
-    "eslint": "^7.5.0",
```

Comment:
This deletion seems like a mistake?

Reply:
This was intentional: Create-React App already includes an Eslint dependency - having this included in package.json causes errors when running `npm run build` and `npm run start`. Even with the deletion `npm install` should still install eslint? Although we aren't 100% sure.

Reply:
Would be good to sanity check this with `rm -rf node_modules && npm install` and then running the linter to see if it still works as expected.

Reply:
Good call - it still works!